plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1721,3 +1728,35 @@ SecRule REQUEST_FILENAME "@rx /apps/mail/api/mailboxes/[0-9]+$" \
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# Writing an draft email
+# Email subject/body could be anything
+# PUT - Composing an draft email
+# DELETE - Discarding a draft email
+SecRule REQUEST_FILENAME "@rx /apps/mail/api/drafts(?:/[0-9]+|/\{id\})?$" \
```

Comment:
With `id`, isn't there any identifier? The URL would look like this: `/apps/mail/api/drafts/{id}`.

And just to be sure, you want disable those targets for:
- `.../drafts`
- `.../drafts/12345`
- `.../drafts/{id}`

Reply:
Yes, it's meant to match all 3. For whatever reason, sometimes the request will be made to the ``/apps/mail/api/drafts/{id}`` endpoint.